.github/workflows/ci.yml

```diff
@@ -559,98 +574,111 @@ jobs:
+                 || contains(github.event.pull_request.labels.*.name, 'tests:all')
+                 || contains(github.event.pull_request.labels.*.name, 'tests:hive')
+                 }}
+          
+          ignore exclusion if:
+            # Do not use this property outside of the matrix configuration.
+            #
+            # This is added to all matrix entries so they may be conditionally
+            # excluded by adding them to the excludes list with a GHA expression
+            # for this property.
+            # - If the expression evaluates to true, it will never match the a
+            #   actual value of the property, and will therefore not be excluded.
+            # - If the expression evaluates to false, it will match the actual
+            #   value of the property, and the exclusion will apply normally.
+            - false
+          
+          include:
+            # this suite is not meant to be run with different configs
+            - config: default
+              suite: suite-6-non-generic
+              jdk: 11
+            # this suite is not meant to be run with different configs
+            - config: default
+              suite: suite-7-non-generic
+              jdk: 11
+            # this suite is not meant to be run with different configs
+            - config: default
+              suite: suite-8-non-generic
+              jdk: 11
+            # this suite is not meant to be run with different configs
+            - config: default
+              suite: suite-tpcds
+              jdk: 11
+            # this suite is not meant to be run with different configs
+            - config: default
+              suite: suite-oauth2
+              jdk: 11
+            # this suite is not meant to be run with different configs
+            - config: default
+              suite: suite-ldap
+              jdk: 11
+            # this suite is not meant to be run with different configs
+            - config: default
+              suite: suite-compatibility
+              jdk: 11
+            # this suite is designed specifically for apache-hive3. TODO remove the suite once we can run all regular tests on apache-hive3.
+            - config: apache-hive3
+              suite: suite-hms-only
+              jdk: 11
+            # suite-1 does not contain Kerberos/encrypted Hive environments
+            - config: hdp3
+              suite: suite-1
+              jdk: 17
+            # suite-2 contains Kerberos/encrypted Hive environments
+            - config: hdp3
+              suite: suite-2
+              jdk: 17
+          EOF
+      - name: Build PT matrix (all)
+        if: steps.filter.outputs.product-tests == 'true' || contains(github.event.pull_request.labels.*.name, 'tests:all') || contains(github.event.pull_request.labels.*.name, 'tests:all-product')
+        run: |
+          ./.github/bin/build-pt-matrix-from-impacted-connectors.py -v -m .github/test-pt-matrix.yaml -o matrix.json
```

Comment:
This basically only converts the matrix from yaml to json, right? Maybe it's worth to add a comment.